tests/test_products/test_category_routers.py

```diff
@@ -5,26 +5,26 @@
 from src.core.factories import CategoryInputSchemaFactory
 
 
-def test_authenticated_user_can_create_category(
+def test_superuser_can_create_category(
     sync_client: TestClient,
-    auth_headers: dict[str, str],
+    superuser_auth_headers: dict[str, str],
 ):
```

Comment:
when testing with staff role introduced, cases should be tested with the lowest permissions possible to execute certain action. For example, getting all products for unauthenticated user(btw fix it in product routers), updating category for staff user and grantng staff permissions for superuser